site/docs/v1/tech/apis/_tenant-request-body.adoc

```diff
@@ -196,6 +196,8 @@ include::_tenant-generator-length-config-rules.adoc[]
 :generator_type!:
 :generator_display_name!:
 +
+[field]#tenant.externalIdentifierConfiguration.samlv2AuthNRequestIdTimeToLiveInSeconds# [type]#[Integer]# [optional]#Optional# [default]#300# [since]#Available since 1.20.0#::
+The time in seconds that a SAML AuthN request will be eligible for use to authenticate with FusionAuth.
```

Comment:
This Id is used to protected against SAML v2 replay attacks. So before you make a request to the SAML v2 IdP Login endpoint `/api/identity-provider/login` you must call `/api/identity-provider/start` to indicate that you want to call the login endpoint. We hand you back a token, and then this must be used to complete the login. 

So this value is probably better described as:

```
The time in seconds that a SAML AuthN request Id returned by the Start SAML v2 Login Request API will be eligible to be used to complete a SAML v2 Login request. 
```

The tooltip for this field may also be helpful for context:
https://github.com/FusionAuth/fusionauth-app/blob/20aa7a0dc616af32fefb0284e8695a7323aff4e3/web/WEB-INF/messages/admin/tenant/package.properties#L236

For context: 

Start request: 
https://fusionauth.io/docs/v1/tech/apis/identity-providers/samlv2/#start-a-saml-v2-login-request

Complete a request with this Id:
https://fusionauth.io/docs/v1/tech/apis/identity-providers/samlv2/#complete-a-saml-v2-login